packages/app/background-jobs-common/UPGRADING.md

```diff
@@ -0,0 +1,45 @@
+# Upgrading Guide
+
+## Upgrading from `4.0.0` to `5.0.0`
+
+### Description of Breaking Changes
+
+1. **Property Removal**
+  - The deprecated `processCalls` property from `FakeBackgroundJobProcessor` has been removed.
+
+2. **Method Changes in `AbstractBackgroundJobProcessor`**
+  - The method `getActiveQueueIds` has been removed. A new constant method now serves this purpose.
+  - `logJobStarted` and `logJobFinished` methods are no longer public. The SDK now offers basic logging, and there are
+    alternative ways to extend logging if needed.
+  - The method `resolveExecutionLogger` can no longer be overridden. The SDK will now automatically create the 
```

Comment:
what if some library user has different logging needs and e. g. wants to have different fields on child logger or use different logger altogether?

Reply:
I think this is covered on the "migration steps" section, right?

Reply:
kinda, but this is one change I'm slightly torn on, making it protected *might* be useful in some cases, but I don't have strong opinion on this, it's ok to leave it as is

Reply:
I had the same concern at the beginning, but from what I saw we are not using it and I am sure if having it there "just in case" is a good idea so decided to remove it for now

Reply:
If we see it is useful in the future we can put it back easily

Reply:
well, we are not using it because default implementation is custom-tailored to Lokalise, I was more concerned about the users outside of it :D

Reply:
not a problem per se, but the kind of flexibility that I think can be useful. but as I've said, I'm also OK to keep it as is